edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -203,68 +177,64 @@ async Task RefreshTwinAsync()
             {
                 Events.TwinRefreshStart();
 
+                Events.GettingModuleClient();
+                IModuleClient moduleClient = await this.moduleConnection.GetOrCreateModuleClient();
```

Comment:
Does this ever throw?

Reply:
Emm, yeah, this could throw (but should not throw). But the exception should get caught and handled.. 
Any particular concerns here?

Reply:
Nope, just wanted to make sure it wasn't a missed case.